src/components/chatmessagecombinator.tsx

```diff
@@ -154,10 +158,23 @@ const ChatMessageCombinator = ({
             )}
           </div>
         </div>
+
         {calculatedMessages.map((msgs, index) => {
-          const patentMessage = msgs.find(
-            (msg) => msg.subRole === "patent-search",
-          );
+          const patentMessage = msgs.find((msg) => {
+            if (msg.subRole === "patent-search") {
+              console.log("patentmsg", patentMessage);
+              return msg.subRole;
+            }
+          });
```

Comment:
why this have been change to return only subrole ?

Reply:
It written for try to  check the patent search bug but now I removed all extra things and it made as it was before.